.github/workflows/mega-linter.yml

```diff
@@ -0,0 +1,79 @@
+---
+# MegaLinter GitHub Action configuration file
+# More info at https://oxsecurity.github.io/megalinter
+name: MegaLinter
+
+on: [push, pull_request]
+
+env:
+  # Apply linter fixes configuration
+  APPLY_FIXES: none
+  APPLY_FIXES_EVENT: pull_request
+  APPLY_FIXES_MODE: pull_request
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: MegaLinter
+    runs-on: ubuntu-latest
+    steps:
+      # Git Checkout
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      # MegaLinter
+      - name: MegaLinter
+        id: ml
+        # You can override MegaLinter flavor used to have faster performances
```

Comment:
```suggestion
        # You can override the MegaLinter flavor used to improve the performance
```

Reply:
All comments come from the default MegaLinter template for GitHub actions. Can we keep it as is?

Reply:
As you like, I think we should make it clear and readable to others as this code will not be maintained by ML.
It only provides the initial copy and we are stuck with the typos/grammer issues for ever ;P

Reply:
As discussed. I'll cleanup the comments and only keep the important ones (e.g. with links to docs).